### ISSUE TITLE
Inconsistency when using external FMT

### ISSUE THREAD

**Issue by luigino** [gabime/spdlog#1355]
Building spdlog 1.4.2 on openSUSE Tumbleweed with cmake.
Even if the SPDLOG_FMT_EXTERNAL=ON switch is passed, the fmt folder is still being included among the headers (minus the bundled subfolder) and there are [a](https://github.com/gabime/spdlog/blob/v1.x/include/spdlog/formatter.h#L6) [couple](https://github.com/gabime/spdlog/blob/v1.x/include/spdlog/common.h#L38) of references to it.

Furthermore, [SPDLOG_FMT_EXTERNAL](https://github.com/gabime/spdlog/blob/v1.x/include/spdlog/tweakme.h#L73) remains undefined.

Basically, I'm unclear if this is on purpose or if it's possible to package spdlog fully relying on a separate libfmt.

Regards

EDIT: My bad, sorry.

**Comment by KottV** [comment]
Hi,

Does it works for you? I'm building nheko on TW and got missing includes error, seems that SPDLOG_FMT_EXTERNAL is ignored;

> In file included from /usr/include/spdlog/common.h:38,
                 from /usr/include/spdlog/spdlog.h:12,
                 from /home/kv/src/nheko_dev/nheko/src/Logging.h:4,
                 from /home/kv/src/nheko_dev/nheko/src/dialogs/PreviewUploadOverlay.cpp:29:
/usr/include/spdlog/fmt/fmt.h:22:10: fatal error: bundled/core.h: Нет такого файла или каталога
   22 | #include "bundled/core.h"

Нет такого файла или каталога = no such file or dir

**Comment by luigino** [comment]
@KottV you must use the pkgconfig flags, otherwise the relevant macro won't be defined.

**Comment by KottV** [comment]
I see, thanks.
